style.css

```diff
@@ -18,8 +18,15 @@ html {
 }
 
 #tabzilla {
+	position:fixed;
     margin-right: 100px;
 }
+/* Actual upgradation
+*  Adding new style on hovering
+*/
+#tabzilla:hover{
```

Comment:
apologies for my wrong words.And I would like to work further on it.

On Fri, Jan 30, 2015 at 11:34 PM, Josh Matthews <notifications@github.com>
wrote:

> In style.css <https://github.com/jdm/asknot/pull/246#discussion_r23859485>
> :
>
> >      margin-right: 100px;
> >  }
> > +/* Actual upgradation
> > +*  Adding new style on hovering
> > +*/
> > +#tabzilla:hover{
>
> How does it expand the font? It's just changing the color property.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jdm/asknot/pull/246/files#r23859485>.
>